### ISSUE TITLE
Engage Administrator Component 3.0.1 causes my subpages to look strange

### ISSUE THREAD

**Issue by TheStingPilot** [akeeba/engage#156]
![2022-01-08 00_05_38-Your IT administrator did not make any apps available to you  en nog 3 andere pa](https://user-images.githubusercontent.com/67457660/148618472-e5503f50-0fa2-428e-bd8a-7bb238fce9d3.png)
Please read the README.md in the repository's root and the support resources before submitting an issue.

### Steps to reproduce the issue

All my articles on my website have been - sort of - destroyed. The pictures cannot be found and the template part is at the bottom part of the website.

### Expected result

A good working article.

### Actual result

The layout has become corrupted. See attachment
![2022-01-08 00_02_36-](https://user-images.githubusercontent.com/67457660/148618300-46642054-1294-4ac7-84b4-f39407028200.png)

### Troubleshooting already performed

If I disable Engage Administrator Component v3.0.1 the layout is perfectly fine. Switching themes does not resvolve this issue.

#### Mandatory information

Issues without this information can not and _will not_ be replied to.

* **Joomla! version**: 4.0.5
* **PHP version**: 7.4.26
* **Am I using HTTPS**: Yes
* **Server Operating System**: Linux
* **My device's Operating System**: Windows, Android
* **My device's Operating System version**: Windows 10
* **Browser**: Chrome, Edge, 
* **Browser version**: x.y.z

#### Good to have information

-

* **MySQL version**: 10.3.27-MariaDB-cll-lve
* **Server type**: Apache/2
* **Server version**: I have no idea.
* **Hosting company: direct-deploy.eu
* **Server Operating System version**: Linux 3.10.0-962.3.2.lve1.5.60.el7.x86_64

**Comment by nikosdion** [comment]
I will be releasing 3.0.2 today to address that. This happens only on _some_ sites and it took me a while to find a way to reproduce it. Once I could reproduce it I found out and fixed what is causing it. By that time it was 10pm and I _never_ do a release before going to bed or otherwise become unavailable for hours — the one time I did poop hit the fan :)

So, I am closing this issue because it's already resolved and the 3.0.2 release will include the fix.
